### PR TITLE
move draft from word to rst

### PR DESCRIPTION
**Pull request by aflaxman**

### CODE REVIEW

**Comment by alibow** [comment]
Would it be useful to have a table of contents for the rest of the cause model guide document in this section? Otherwise it looks good to me!

**Comment by collijk** [comment]
Text wrapped at 80 chars to make reading in a text editor easier.  Fixed some thing to make the rst render properly into html.

**Comment by collijk** [comment]
See https://github.com/ihmeuw/vivarium_research/pull/11 for cross links and glossary stuff to come into this PR.

File: docs/source/models/cause.rst [hunk 1]
```diff
@@ -4,17 +4,119 @@
...
+the scope of this chapter, because they do not fit into the Markov approach.
+This chapter does not include “PAF of 1” causes, like protein-energy
+malnutrition (PEM).  It also does not include things that are sometimes
+considered a disease but are deemed “risk factors” by GBD, such as obesity.
+Unsafe water would not fit into the cause model paradigm either---it is a cause
+of diarrheal disease, but it is a risk factor in the GBD taxonomy.  Latent
+tuberculosis infection (LTBI) does fit into this chapter, but just barely.
 
 .. contents:
+   :local:
```
Comment (collijk) [inline docs/source/models/cause.rst]: @alibow This renders a table of contents.  The raw docs are like source code.  Many really cool and important features won't show up until we render them at https://vivarium-research.readthedocs.io/en/latest/

If you follow steps in the contribution guide, you can render them locally.  I always try to review docs on my own computer looking at the HTML page.  I'll show everyone how to do this on Friday.

File: docs/source/models/cause.rst [hunk 2]
```diff
@@ -4,17 +4,119 @@
...
+Short answer: disease
+
+More nuance: “cause of death” as might be included as the bottom line of the
+top half of a death certificate. And also analogous causes of nonfatal health
+loss, too.
+
+.. image:: death_certificate.png
+
+
+Learning objectives
```
Comment (collijk) [inline docs/source/models/cause.rst]: This seems like it should be the first subsection

File: docs/source/models/cause.rst [hunk 3]
```diff
@@ -4,17 +4,119 @@
...
+considered a disease but are deemed “risk factors” by GBD, such as obesity.
+Unsafe water would not fit into the cause model paradigm either---it is a cause
+of diarrheal disease, but it is a risk factor in the GBD taxonomy.  Latent
+tuberculosis infection (LTBI) does fit into this chapter, but just barely.
 
 .. contents:
+   :local:
+
+
+What is a "cause"?
```
Comment (collijk) [inline docs/source/models/cause.rst]: I think having a combined What is a cause/what is a cause model section might be more helpful.  A lot of the work in producing a good cause model is bridging that gap, so a compare/contrast might be helpful.  It also seems like there are three levels to this:

- The cause in the world (ie the thing we want to model)
- The GBD model(s) of the cause (fatal, nonfatal, intermediate things like dismod models).
- The **cause model**, ie the vivarium model of the cause.

File: docs/source/models/cause.rst [hunk 4]
```diff
@@ -4,17 +4,119 @@
...
+complexity of reality, and is designed to be acceptable to outside experts on
+the cause, as well as be parsimonious with the data available from the GBD that
+might inform the model.
+
+
+How is a cause model incorporated into a larger model?
+------------------------------------------------------
+
+Our modular structure is designed to layer cause models into the
+entity-component system that has a demographic model.  Sometimes an
```
Comment (collijk) [inline docs/source/models/cause.rst]: I'm uncertain how much a reference to a software pattern is helpful here.  

I also think splitting this cleanly into two ideas is helpful:
- How does a cause model affect other parts of the model.
- How is a cause model affected by other parts of the model.

File: docs/source/models/cause.rst [hunk 5]
```diff
@@ -4,17 +4,119 @@
...
+intervention model will be layered in on top of this and directly change
+transition rates in one or more cause models.  But to date, it has been more
+common to have one or more risk factor models layered in to affect the
+incidence rates in the cause model, and then have an intervention model shift
+the risk exposure levels defined by the risk factor model.
+
+
+Focus on goal 3:
```
Comment (collijk) [inline docs/source/models/cause.rst]: Unneccesary heading

File: docs/source/models/cause.rst [hunk 6]
```diff
@@ -4,17 +4,119 @@
...
-The structure of a cause model
-------------------------------
+
+Why do we want a document that describes each cause model?
+----------------------------------------------------------
+
+* Because a lot of work goes into gaining understanding and developing an
+  appropriately complex model, and we don’t want to repeat that work.
+* Because we (researchers) need to communicate clearly and precisely with
+  software engineers and data scientists about what the model must do and what
```
Comment (collijk) [inline docs/source/models/cause.rst]: and with each other!

File: docs/source/models/cause.rst [hunk 7]
```diff
@@ -4,17 +4,119 @@
...
+  data must inform it.
+* Because we will need to communicate to an outside audience, including
+  critics, how we generated substantive results of interest, and that will
+  include readers who want to know exactly how we modeled the diseases included
+  in our work.
+
+
+What does a model document look like?
+-------------------------------------
```
Comment (collijk) [inline docs/source/models/cause.rst]: We should probably spend a whole session on what the cause model template should look like and then build an rst file out of it.

File: docs/source/models/cause.rst [hunk 8]
```diff
@@ -4,17 +4,119 @@
...
+   challenges.
+
+
+What is a cause model?
+----------------------
+
+A cause model is a simplification of a cause of death or disease for the
+purposes of simulation modeling.  It is always an idealization of the messy
+complexity of reality, and is designed to be acceptable to outside experts on
+the cause, as well as be parsimonious with the data available from the GBD that
```
Comment (collijk) [inline docs/source/models/cause.rst]: Parsimonious is a weird word to use here.

File: docs/source/models/cause.rst (hunk 1 above)
Comment (alibow) [reply docs/source/models/cause.rst]: Thanks James! Sorry for the confusion.

File: docs/source/models/cause.rst (hunk 2 above)
Comment (alibow) [reply docs/source/models/cause.rst]: I agree!

File: docs/source/models/cause.rst (hunk 7 above)
Comment (alibow) [reply docs/source/models/cause.rst]: I agree. I think a cause model template document would be more helpful than this list.

**Comment by aflaxman** [comment]
@alibow take a look at this, i think i can merge it if you approve

**Pull request merged**